lib/galaxy/config/sample/datatypes_conf.xml.sample

```diff
@@ -1017,6 +1017,7 @@
     <datatype extension="fits" type="galaxy.datatypes.binary:FITS" mimetype="application/octet-stream" display_in_upload="true" description="Flexible Image Transport System (FITS) used in Astronomy"/>
     <datatype extension="chain" type="galaxy.datatypes.chain:Chain" display_in_upload="true" description_url="https://genome.ucsc.edu/goldenPath/help/chain.html"/>
     <datatype extension="ucsc.net" type="galaxy.datatypes.chain:Net" display_in_upload="true" description_url="https://genome.ucsc.edu/goldenPath/help/net.html"/>
+    <datatype extension="bcsp" type="galaxy.datatypes.binary:bcsp" display_in_upload="true"/>
```

Comment:
You can do it similar to here, just subclass the Binary datatype.

> <datatype extension="bil" type="galaxy.datatypes.binary:Binary" mimetype="application/octet-stream" display_in_upload="true" subclass="true" description="ENVI file with band interleave by line (BIL) format"/>

Reply:
https://github.com/galaxyproject/galaxy/pull/19363/commits/9ead291dadd50c3240d0b773156eb88c15a19fd8